docs/products/kafka/reference/kafka-metrics-prometheus.md

```diff
@@ -2,518 +2,546 @@
+
+## Memory Usage
+
+Metrics starting with `java_lang_Memory` provide insights into the JVM's memory usage,
+including committed memory, initial memory, max memory, and used memory.
+
+| Metric                                             | Description                                                                         |
+|----------------------------------------------------|-------------------------------------------------------------------------------------|
+| `java_lang_Memory_committed`                       | Returns the amount of memory in bytes that is committed for the Java virtual machine to use.  |
+| `java_lang_Memory_init`                            | Returns the amount of memory in bytes that the Java virtual machine initially requests from the operating system for memory management. |
+| `java_lang_Memory_max`                             | Returns the maximum amount of memory in bytes that can be used for memory management.  |
+| `java_lang_Memory_used`                            | Returns the amount of used memory in bytes.                                          |
+| `java_lang_Memory_ObjectPendingFinalizationCount`  | Number of objects pending finalization.                                              |
 
 ## Apache Kafka Connect
 
-The Apache Kafka Connect metrics list is available in the
-[dedicated page](../kafka-connect/reference/connect-metrics-prometheus).
+For a comprehensive list of Apache Kafka Connect metrics exposed through Prometheus,
+see [Apache Kafka® Connect available via Prometheus](../kafka-connect/reference/connect-metrics-prometheus).
 
-## Apache Kafka broker
+## Apache Kafka broker metrics
 
-The descriptions for the below metrics are available in the [Monitoring
-section of the Apache Kafka
-documentation](https://kafka.apache.org/documentation/#monitoring).
+Apache Kafka brokers expose metrics that provide insights into the health and
+performance of the Apache Kafka cluster. Find detailed descriptions of these metrics,
+see the [monitoring section of the Apache Kafka documentation](https://kafka.apache.org/documentation/#monitoring).
 
-:::note
-The metrics with a `_Count` suffix are cumulative counters for the given
-metric, for example,
-`kafka_controller_ControllerStats_LeaderElectionRateAndTimeMs_Count`.
+### Metric types
 
-Note that a metric like
-`kafka_server_BrokerTopicMetrics_MessagesInPerSec_Count` is a cumulative
-count of incoming messages despite the `PerSec` suffix in the metric
-name.
+#### Cumulative counters (`_count`)
+Metrics with a `_count` suffix are cumulative counters. They track the total number of
+occurrences for a specific event since the broker started.
 
-To see the rate of change of these `_Count` metrics, you can apply a
-function such as the `rate()` function in PromQL.
-:::
+**Example:**
+- `kafka_controller_ControllerStats_LeaderElectionRateAndTimeMs_Count`: Total number of
```

Comment:
For this example and the one below, I'd remove an unordered list coz there is one example only in each case.
```suggestion
`kafka_controller_ControllerStats_LeaderElectionRateAndTimeMs_Count`: Total number of
```